Documentation/guides/messages/xa5205.md

```diff
@@ -1,26 +1,22 @@
 ---
 title: Xamarin.Android error XA5205
 description: XA5205 error code
-ms.date: 05/03/2018
+ms.date: 02/14/2020
 ---
 # Xamarin.Android error XA5205
 
 ## Example messages
 
 ```
-XA5205: The Android SDK Directory could not be found. Please set via `/p:AndroidSdkDirectory`.
+error XA5205: Cannot find `aapt.exe`. Please install the Android SDK Build-Tools package with the `C:\Program Files (x86)\Android\android-sdk\tools\android.bat` program.
```

Comment:
I considered updating the installer tool name from `android` to `sdkmanager` to account for the deprecation of the `android` tool in Android SDK Tools version 25.3.0 and higher (<https://developer.android.com/studio/releases/sdk-tools>).

But the Visual Studio Android SDK Manager still offers an option to switch back to Android SDK Tools version 25.2.5, and on new SDK Tools versions, the deprecated `android` command provides an informational message and tries to run `sdkmanager` for the user, so leaving `android` as the tool name seemed like it might still be the easiest option for the moment.